provisionersdk/proto/provisioner.proto

```diff
@@ -141,6 +141,7 @@ message Agent {
 	repeated Env extra_envs = 22;
 	int64 order = 23;
 	ResourcesMonitoring resources_monitoring = 24;
+	repeated Devcontainer devcontainers = 25;
```

Comment:
Even though the RPC protocol defined in this file is not versioned, this message is imported by `provisionerd.proto`, which _is_ versioned and so we need to bump the minor version.